pkg/cmd/agent-task/view/view.go

```diff
@@ -317,6 +317,28 @@ func printSession(opts *ViewOptions, session *capi.Session) {
 
 	fmt.Fprintf(opts.IO.Out, "%s%s\n", cs.Muted(usedPremiumRequestsNote), cs.Muted(durationNote))
 
+	if session.Error != nil {
+		var workflowRunURL string
+		if session.WorkflowRunID != 0 && session.PullRequest != nil {
+			if u, err := url.Parse(session.PullRequest.URL); err == nil {
+				workflowRunURL = fmt.Sprintf("%s://%s/%s/actions/runs/%d", u.Scheme, u.Host, session.PullRequest.Repository.NameWithOwner, session.WorkflowRunID)
+			}
+		}
+
+		message := session.Error.Message
+		if message == "" {
+			message = "An error occurred"
+		}
+		fmt.Fprintln(opts.IO.Out, "")
+		fmt.Fprintf(opts.IO.Out, "%s %s\n", cs.FailureIconWithColor(cs.Red), message)
```

Comment:
Why not add a `\n` to prefix the string instead?

```suggestion
		fmt.Fprintf(opts.IO.Out, "\n%s %s\n", cs.FailureIconWithColor(cs.Red), message)
```

Reply:
Done in next push.